src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -90,7 +90,8 @@ object CheckHighForm extends Pass {
           correctNum(Option(1), 2)
           val (msb, lsb) = (e.consts(0).toInt, e.consts(1).toInt)
           if (lsb > msb) {
-            errors.append(new LsbLargerThanMsb(info, mname, e.op.toString, lsb, msb))
+            errors.append(
```

Comment:
Nit: If the line limit is 100 characters you don't need a line break.

Reply:
I'm in agreement. This is my internal >80 alarm going off. I need to setup per-project settings...